server/src/main/java/com/networknt/server/Server.java

```diff
@@ -125,6 +123,8 @@ public static void init() {
         try {
             // load config files from light-config-server if possible.
             loadConfig();
+            // merge status.yml and app-status.yml if app-status.yml is provided
+            mergeStatusConfig();
```

Comment:
just as I'm new to light-4j, I don't know that 'status' is a mandatory module for light-4j server or not? where we configure that?
If not, what happens if user doesn't use 'status' module?

Reply:
@lanphan Yes. status is a mandatory module as it contains all the error code and description for the error on a particular server. For more info, you can visit https://doc.networknt.com/concern/status/

There is a status.yml in the status module itself and it contains all the status code for the framework, the end user can externalized it and add application specific error codes to it. Or they can just add an app-status.yml which contains only the application specific status codes. This module is integrated with the server and almost every handler, so it is part of the server instance and cannot be skipped.